src/planwise/client/projects2/components/dashboard.cljs

```diff
@@ -8,11 +8,19 @@
             [planwise.client.routes :as routes]
             [planwise.client.ui.common :as ui]
             [planwise.client.ui.rmwc :as m]
-            [planwise.client.utils :as utils]))
+            [planwise.client.utils :as utils]
+            [planwise.client.projects2.components.settings :as settings]))
 
 (defn- project-tabs
-  [{:keys [active] :or {active 0}}]
-  [m/TabBar {:activeTabIndex active}
+  [{:keys [active] :or {active :scenarios}}]
+  [m/TabBar {:activeTabIndex (cond
```

Comment:
You may want to use `case` here instead (when all the conditions are simple equals), or even a map, which can act as functions on the keys:

```clojure
({:scenarios 0 :settings 1} active)
```